src/common/sorted-table/index.js

```diff
@@ -666,6 +686,15 @@ export default class SortedTable extends Component {
                 userData={userData}
               />
             ))}
+            {map(actions, (props, key) => (
+              <IndividualAction
+                {...props}
+                handler={props.individualHandler || props.handler}
+                item={props.individualHandler !== undefined ? item : [item]}
+                key={key + individualActions.length}
```

Comment:
Actually, [it doesn't seem to be necessary](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings).